gm4_standard_crafting/data/gm4_standard_crafting/functions/recipe_check.mcfunction

```diff
@@ -6,13 +6,20 @@ execute if score @s gm4_slot_count matches 1 if score @s gm4_stack_size matches
 execute if score @s gm4_slot_count matches 1 if score @s gm4_stack_size matches ..16 if data storage gm4_custom_crafters:temp/crafter {Items:[{Slot:0b,id:"minecraft:chiseled_red_sandstone"}]} run data merge storage gm4_custom_crafters:temp/crafter {Items:[{Slot:8,id:"red_sand",Count:1b,tag:{gm4_custom_crafters:{multiplier:4}}}]}
 execute if score @s gm4_slot_count matches 1 if score @s gm4_stack_size matches ..16 if data storage gm4_custom_crafters:temp/crafter {Items:[{Slot:0b,id:"minecraft:cut_red_sandstone"}]} run data merge storage gm4_custom_crafters:temp/crafter {Items:[{Slot:8,id:"red_sand",Count:1b,tag:{gm4_custom_crafters:{multiplier:4}}}]}
 execute if score @s gm4_slot_count matches 1 if score @s gm4_stack_size matches ..16 if data storage gm4_custom_crafters:temp/crafter {Items:[{Slot:0b,id:"minecraft:smooth_red_sandstone"}]} run data merge storage gm4_custom_crafters:temp/crafter {Items:[{Slot:8,id:"red_sand",Count:1b,tag:{gm4_custom_crafters:{multiplier:4}}}]}
+
+#sandstone slabs to sand
+execute if score @s gm4_slot_count matches 1 if score @s gm4_stack_size matches ..32 if data storage gm4_custom_crafters:temp/crafter {Items:[{Slot:0b,id:"minecraft:sandstone_slab"}]} run data merge storage gm4_custom_crafters:temp/crafter {Items:[{Slot:8,id:"sand",Count:1b,tag:{gm4_custom_crafters:{multiplier:2}}}]}
+execute if score @s gm4_slot_count matches 1 if score @s gm4_stack_size matches ..32 if data storage gm4_custom_crafters:temp/crafter {Items:[{Slot:0b,id:"minecraft:cut_sandstone_slab"}]} run data merge storage gm4_custom_crafters:temp/crafter {Items:[{Slot:8,id:"sand",Count:1b,tag:{gm4_custom_crafters:{multiplier:2}}}]}
+execute if score @s gm4_slot_count matches 1 if score @s gm4_stack_size matches ..32 if data storage gm4_custom_crafters:temp/crafter {Items:[{Slot:0b,id:"minecraft:smooth_sandstone_slab"}]} run data merge storage gm4_custom_crafters:temp/crafter {Items:[{Slot:8,id:"sand",Count:1b,tag:{gm4_custom_crafters:{multiplier:2}}}]}
+
 execute if score @s gm4_slot_count matches 1 if score @s gm4_stack_size matches ..32 if data storage gm4_custom_crafters:temp/crafter {Items:[{Slot:0b,id:"minecraft:red_sandstone_slab"}]} run data merge storage gm4_custom_crafters:temp/crafter {Items:[{Slot:8,id:"red_sand",Count:1b,tag:{gm4_custom_crafters:{multiplier:2}}}]}
+execute if score @s gm4_slot_count matches 1 if score @s gm4_stack_size matches ..32 if data storage gm4_custom_crafters:temp/crafter {Items:[{Slot:0b,id:"minecraft:cut_red_sandstone_slab"}]} run data merge storage gm4_custom_crafters:temp/crafter {Items:[{Slot:8,id:"red_sand",Count:1b,tag:{gm4_custom_crafters:{multiplier:2}}}]}
+execute if score @s gm4_slot_count matches 1 if score @s gm4_stack_size matches ..32 if data storage gm4_custom_crafters:temp/crafter {Items:[{Slot:0b,id:"minecraft:smooth_red_sandstone_slab"}]} run data merge storage gm4_custom_crafters:temp/crafter {Items:[{Slot:8,id:"red_sand",Count:1b,tag:{gm4_custom_crafters:{multiplier:2}}}]}
```

Comment:
Do we even need these? You can craft `sandstone_slab` to `sandstone` and then to `sand` already.

Reply:
I was just trying to make it consistent; slabs were there before slab -> block was added. This is unnecessary now tho